src/displayTodos.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable */
+import { Storage } from './storage';
+import { TodoDOMElement } from './classDom';
+
+const todosContainer = document.querySelector('.todos-container');
+
+const showTodos = (index = 0) => {
+  const todos = Storage.getTodos(index);
+  todosContainer.innerHTML = '';
+  projectReference(index);
+
+  if (todos.length > 0) {
+    todos.forEach((todo, todoIndex) => {
+      const todoEl = new TodoDOMElement(
+        todo.title,
+        todo.due,
+        todo.priority,
+        todoIndex,
+        todo.done,
+      );
+      todosContainer.appendChild(todoEl.div);
```

Comment:
Will like to see you're using the destructuring object, I'll give you an example here so you can apply it everywhere
```javascript
todos.forEach((todo, todoIndex) => {
      const {title, due, priority, done} = todo
      const todoEl = new TodoDOMElement(
        title,
        due,
        priority,
        todoIndex,
        done,
      );
      todosContainer.appendChild(todoEl.div);
});
```